cmd/vcfInfo/testdata/test.vcf

```diff
@@ -0,0 +1,30 @@
+chr13	113593242	13:113593242:G:A	G	A	255	PASS	AA=G	GT	0|0	0|0	0|0	0|0	0|0	0|0
```

Comment:
Is everything in the ID field just for debug?

Reply:
Arbitrary ID data, just copied from another test file elsewhere in the code base.